src/cml.js

```diff
@@ -69,8 +70,41 @@ const getDriver = (opts) => {
   throw new Error(`driver ${driver} unknown!`);
 };
 
+const fixGitSafeDirectory = () => {
+  const getOrSetGitConfigSafeDirectory = (value) =>
```

Comment:
not a big fan of combining get and set this way - saves some space (?), but make readability worse

Also, usually `getOrSet` would take a value and would set value iff it's not set already, it also makes this `get w/o value, set with value` interface confusing and unexpected

Reply:
I wholeheartedly agree. I wrote it that way to avoid maintaining two versions of an almost identical `git` command, but it's really ugly.

Reply:
Resolving in favor of https://github.com/iterative/cml/issues/984

Reply:
Resolved with #986